core/templates/dev/head/domain/exploration/StateObjectFactory.js

```diff
@@ -19,36 +19,47 @@
 
 oppia.factory('StateObjectFactory', [
   'AnswerGroupObjectFactory', 'InteractionObjectFactory',
-  function(AnswerGroupObjectFactory, InteractionObjectFactory) {
-  var State = function(name, classifierModelId, contentBackendList,
-    interactionBackendDict, paramChangesBackendList) {
-    this.name = name;
-    this.classifierModelId = classifierModelId;
-    this.content = contentBackendList;
-    this.interaction = InteractionObjectFactory.create(interactionBackendDict);
-    this.paramChanges = paramChangesBackendList;
-  };
+  'ContentObjectFactory',
+  function(AnswerGroupObjectFactory, InteractionObjectFactory,
+    ContentObjectFactory) {
+    var State = function(name, classifierModelId, contentBackendList,
```

Comment:
Well actually nvm there is no conversion in AnswerGroupObjectFactory yet so nothing would change